packages/i/i3-style/package.yml

```diff
@@ -12,8 +12,10 @@ description: |
     Make your i3 config a little more stylish.
 builddeps  :
     - rust
+setup   : |
+    cargo fetch
```

Comment:
Should be `%cargo_fetch` :)

Reply:
I've been running into an issue using the macro, it runs as cargo fetch --locked and I get this error: 
`error: the lock file /home/build/YPKG/root/i3-style/build/i3-style-1.0.2/Cargo.lock needs to be updated but --locked was passed to prevent this
If you want to try to generate the lock file without accessing the network, remove the --locked flag and use --offline instead.`
The --offline flag doesn't resolve it either. Should I create an updated lock file separately and patch that in to resolve it?